exercises/practice/square-root/test/square_root_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule SquareRootTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "root of 1" do
+    radicand = 1
+    output = SquareRoot.square_root(radicand)
```

Comment:
"square root square root" :D sounds a bit silly. Maybe we can deviate from problem specs in this case and do, let's say `SquareRoot.calculate`?

Reply:
☝🏻

Reply:
It might look better in the tests, but it will look weird in the lib file. How about `sqrt`?

Reply:
My problem with `SquareRoot.square_root` and `SquareRoot.sqrt` is that in both cases both the module and the function name are exactly the same noun, shortened or not. I don't really know why, but it feels more correct to me if function name is either an action that gets executed on the "subject" indicated by the module name, or if the function name is a noun that's a specialization of the module name. I wonder if there's an article about this 🤔 I would like to know where my feeling comes from, maybe it's completely unjustified?

Here's more proposals. They all look ok to me 🤷 

```elixir
defmodule SquareRoot do
  @doc """
  Calculate the integer square root of a positive integer
  """
  @spec calculate(radicand :: pos_integer) :: pos_integer
  def calculate(radicand) do
  end
end
```
```elixir
defmodule Root do
  @doc """
  Calculate the integer square root of a positive integer
  """
  @spec square(radicand :: pos_integer) :: pos_integer
  def square(radicand) do
  end
end
```
```elixir
defmodule Math do
  @doc """
  Calculate the integer square root of a positive integer
  """
  @spec sqrt(radicand :: pos_integer) :: pos_integer
  def sqrt(radicand) do
  end
end
```
```elixir
defmodule Calculator do
  @doc """
  Calculate the integer square root of a positive integer
  """
  @spec sqrt(radicand :: pos_integer) :: pos_integer
  def sqrt(radicand) do
  end
end
```
```elixir
defmodule PosInteger do
  @doc """
  Calculate the integer square root of a positive integer
  """
  @spec sqrt(radicand :: pos_integer) :: pos_integer
  def sqrt(radicand) do
  end
end
```